code/modules/crafting/recipes.dm

```diff
@@ -578,15 +578,15 @@
 	result = /obj/item/decorations/sticky_decorations/flammable/heart
 	reqs = list(/obj/item/paper = 1,
 				/obj/item/stack/tape_roll = 1)
-	tools = list(/obj/item/wirecutters,
+	tools = list(TOOL_WIRECUTTER,
```

Comment:
I'd make a new list for the "other tools".
Call it extra_required_items or such.
This will keep the list of one type and will be easier to understand later on for people

Reply:
it's fixable enough.